03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const connectDB = (url) => {
+    mongoose.connect(url, { // second argument is an object with options
```

Comment:
> second argument is an object with options

Why might we want to do this instead of using normal arguments to the `mongoose.connect` function?